spec/classes/hourly_spec.rb

```diff
@@ -1,52 +0,0 @@
-require 'spec_helper'
```

Comment:
Can you please elaborate why you removed this test? This class still exists.

Reply:
Is it now private, (and should be marked as such?)

Reply:
@alexjfisher indeed, I forgot to add the `assert_private()`
@baurmatt I have moved the test to init_spec.rb since the class is (now) private and will be handled via the main class.